index.js

```diff
@@ -8,10 +17,33 @@ function extendMemoryDB(MemoryDB) {
 
   ShareDBMingo.prototype = Object.create(MemoryDB.prototype);
 
+  ShareDBMingo.prototype.projectsSnapshots = true;
```

Comment:
We shouldn't need to implement projections if the code in ShareDB is working correctly. By default, ShareDB is supposed to implement projections unless the database adapter chooses to implement them natively. Since this is all being done in memory anyway, we might as well leverage the projection code in ShareDB.

Did you discover that projections were not working?

Reply:
Looks like you're right. Tests relying on projections still pass if all this projection code is removed.

I think I had assumed that `projectsSnapshots = true` was required in order for projections to work, but didn't realize that it mean that it controlled whether the adapter or ShareDB was responsible for doing the actual field projection.

Removing! 👋